src/Middleware/RequestThrottling/src/RequestThrottlingOptions.cs

```diff
@@ -12,7 +12,9 @@ public class RequestThrottlingOptions
     {
         /// <summary>
         /// Maximum number of concurrent requests. Any extras will be queued on the server. 
+        /// We do not provide a default since reasonable values can vary from 500 to 500 thousand.
```

Comment:
There's no `we` here. This sounds like more of a regular `//` comment to devs, not a comment to an end user. In what way would this help them?

Reply:
This is null by default because the correct value is application specific. This option must be configured by the application.